src/main/java/core/basesyntax/service/DataConverterImpl.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+
+    public static final int TYPE = 0;
+    public static final int FRUIT = 1;
+    public static final int QUANTITY = 2;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputReport) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : inputReport) {
+            String[] words = line.split(",");
+            String type = words[TYPE];
+            String fruit = words[FRUIT];
+            String quantity = words[QUANTITY];
+            switch (type) {
```

Comment:
The usage of switch-case for operation handling is discouraged in the task description. Instead, you should implement the Strategy pattern to handle different operations.